src/dispatch/case/models.py

```diff
@@ -17,9 +17,9 @@
 from sqlalchemy_utils import TSVectorType
 
 from dispatch.auth.models import UserRead
-from dispatch.case.priority.models import CasePriorityRead
-from dispatch.case.severity.models import CaseSeverityRead
-from dispatch.case.type.models import CaseTypeRead
+from dispatch.case.priority.models import CasePriorityRead, CasePriorityBase
+from dispatch.case.severity.models import CaseSeverityRead, CaseSeverityBase
+from dispatch.case.type.models import CaseTypeRead, CaseTypeBase
```

Comment:
```suggestion
from dispatch.case.priority.models import (
    CasePriorityCreate,
    CasePriorityRead,
    CasePriorityBase,
)
from dispatch.case.severity.models import CaseSeverityBase, CaseSeverityCreate, CaseSeverityRead
from dispatch.case.type.models import CaseTypeBase, CaseTypeCreate, CaseTypeRead
```

`CaseCreate` seems to be using `Read` models and is causing issues on case creation flow for me via Slack Shortcut reports.

```python
class CaseCreate(CaseBase):
    assignee: Optional[UserRead]
    case_priority: Optional[CasePriorityCreate]
    case_severity: Optional[CaseSeverityCreate]
    case_type: Optional[CaseTypeCreate]
    project: Optional[ProjectRead]
    tags: Optional[List[TagRead]] = []
```

Reply:
Can you explain the issue? Is it because the `read` models are expecting an `id` and we only pass in `name`?